benchmarks/src/bench/mod.rs

```diff
@@ -124,3 +130,34 @@ pub async fn run_benchmark<B: Benchmark + Send + Sync>(
     test_durations.sort();
     Ok(test_durations)
 }
+
+pub async fn drop_database(uri: &str, database: &str) -> Result<()> {
```

Comment:
this was needed to avoid running into `NamespaceNotFound` issues in the benchmarks on sharded clusters. See the link in the comment below for more info.